_docs/user-created.md

```diff
@@ -0,0 +1,33 @@
+---
+title: User Created
+permalink: /docs/extras/user-created/
+---
+# User Created
+## User Created Projects
+
+Of course, the Dockstore team is not the only ones that write code or documentation to work with CWL tools and Dockstore. Here we list some user-created projects that make it easier to work with Dockstore in various ways
+
+## [cgp-dockstore](https://github.com/cancerit/cgp-dockstore)
+
+Out of the Cancer Genome Project at the Wellcome Trust, Sanger Institute, this project makes it easier for users that are familiar with [Packer](https://www.packer.io/) to work with tools and images registered on Dockstore on OpenStack.
+
+## [PCAWG-Docker-Test](https://github.com/mikisvaz/PCAWG-Docker-Test)
+
+Miguel Vazquez has been leading the docktesters group. This is a group that is in charge of testing the [PCAWG](http://icgc.org/working-pancancer-data-aws)(PanCancer Analysis of Whole Genomes) tools and workflows. As a part of this work he created a series of scripts (linked above) that demonstrate how to get data and run through all of the PCAWG analysis in sequence.  
+
+## [Alignment pipeline tutorial for the Pan Prostate project](https://docs.google.com/document/d/1djQ-8YlqfjbOJRnkeCu7lV1he_vJygIXn4ExHz6m874/edit?usp=sharing)
+
+Tutorials and documentation are always appreciated as well!
+The following tutorial demonstrates how to run the Pan Prostate pipeline and how to work with Dockstore tools (normally using Docker images) using Singularity.
+
+## Use Case Posters
+
+As Dockstore gains traction people are showing how they are using it in posters
+for various conferences.  Feel free to create a fork and add yours.
+
+### 2017
+#### BOSC
+* [CGP as a Service: From data submission to results using your web-browser](https://github.com/keiranmraine/dockstore-ui/raw/7b18a9ccf4a853088c677e463f8f9cd924b6533c/app/docs/markdown/user_posters/2017_BOSC_Sanger_keiranmraine.pdf)
```

Comment:
This url location looks like it might go away at some point, let's just store it in the gh-page branch

Reply:
For both links or just the CGP one?

Reply:
I think this one is a fork of us which I think is more likely to be deleted. The other seems to be more "original", so I'm less worried about that one